packages/xod-patch-search/test/index.spec.js

```diff
@@ -0,0 +1,89 @@
+import path from 'path';
+import { assert } from 'chai';
+import { listPatches } from 'xod-project';
+import { loadProject } from 'xod-fs';
+
+import { createIndex, createIndexData } from '../src/index';
+
+describe('xod-patch-search/index', () => {
+  let idx = {};
+  const workspace = path.resolve(__dirname, '../../../workspace');
+  const getProjectPath = projectName => path.resolve(workspace, projectName);
+
+  before(async () => {
+    const project = await loadProject(getProjectPath('welcome-to-xod'));
+    const indexData = createIndexData(listPatches(project));
+    idx = createIndex(indexData);
+  });
+
+  it('searches by path correctly', () => {
+    const result = idx.search('path:number');
```

Comment:
Wow. If that works, should we really have `lib:` syntax? Maybe just `path:` is enough?

Reply:
Actually, I'm not sure that is working really correct. But for our current dataset, it returns a correct result. I think that will be good to extract tokens like this from a query and filter data using our algorithms.
But for now, it's working good enough. So I propose to do it later when it will be really needed.